WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -275,6 +277,25 @@ public SendTabViewModel(WalletViewModel walletViewModel, bool isTransactionBuild
 							return;
 						}
 					}
+
+					decimal customFee = 0;
+					if (IsCustomFee)
+					{
+						if (decimal.TryParse(UserFeeText.Replace(',', '.'), out customFee))
+						{
+							if (customFee < 1)
```

Comment:
https://bitcoin.stackexchange.com/questions/48235/what-is-the-minrelaytxfee

Reply:
Free transactions are not allow anymore. However, the lower bound limits `MinRelayTxFee` and `MempoolMinFee` are expressed in satoshies per kilobyte (by default 1000 sat/kb) what means that a zero fee transaction will not go to far. However, I am not sure it is our job to enforce these limits. If I want to build a zero fee transaction why should Wasabi stop me?
I think we should alert the user when the fee is out of some reasonable limits [min_safe_fee, max_safe_fee] but no stop the user.

Reply:
I implemented allowing fees less than 1 sat per byte, but the `TransactionBuilder.check()` will throw an error [here](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi/Services/WalletService.cs#L1266) because the fee is too low.

Reply:
Ok if it cannot be done than let 1 to be the minimum.